qase-cucumberjs/src/reporter.ts

```diff
@@ -247,10 +247,10 @@ export class CucumberQaseReporter extends Formatter {
             stacktrace: error?.stack ?? null,
             thread: null,
           },
-          fields: new Map<string, string>(),
+          fields: {},
```

Comment:
Looks valid, but is it related to test parameterisation?

Reply:
I changed the field type for parameters and custom fields